manifoldco/ui
# Add owner-id helper

## Reason for change

<img width="524" alt="Screen Shot 2019-09-18 at 11 21 32" src="https://user-images.githubusercontent.com/1369770/65170728-7fc84100-da06-11e9-99cc-590899a1378b.png">

This is a small timesaver for the resource **provision** story.

Before, in order to test **provision**, you had to go look up your own user ID, which was a pain. This adds a call to Manifold’s identity service to grab your user ID for you automatically from your token, and save it in localStorage.

## Testing

1. Add a valid token to **manifold_api_token** from the Manifold Dashboard in Storybook. Refresh.
1. Go to the **Resource > provision** story in Storybook, and see that the `owner-id` is set for you (may take a second refresh if you start out on the provision story).

## Checklist




[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform
